lib/macho/load_commands.rb

```diff
@@ -171,6 +27,33 @@ def self.new_from_bin(view)
       self.new(view, *bin.unpack(format))
     end
 
+    # Creates a new (viewless) command corresponding to the symbol provided
+    # @param cmd_sym [Symbol] the symbol of the load command being created
+    # @param args [Array] the arguments for the load command being created
+    # @note this does not currently work. DO NOT USE IT.
+    def self.create(cmd_sym, *args)
+      raise LoadCommandNotCreatableError.new(cmd_sym) unless CREATABLE_LOAD_COMMANDS.include?(cmd_sym)
+
+      klass = MachO.const_get "#{LC_STRUCTURES[cmd_sym]}"
+      cmd = LOAD_COMMAND_CONSTANTS[cmd_sym]
+      cmdsize = klass.bytesize
```

Comment:
I don't think it's wise to set up `cmdsize` in this method. Let's just make it `nil` and delay figuring out `cmdsize` until the load command created here gets serialized. That's the earliest we can know the size, as it can be influenced (among other things) by the required padding that depends on whether we're serializing for a 32-bit or 64-bit Mach-O binary.